app/modules/ads/legacyModule.js

```diff
@@ -122,7 +122,6 @@ class Ads {
 				'ext.wikia.adEngine.adLogicPageParams',
 				'ext.wikia.adEngine.config.mobile',
 				'ext.wikia.adEngine.context.slotsContext',
-				'ext.wikia.adEngine.lookup.a9',
```

Comment:
We can't remove that :(. Also we need to handle fvDelay on mobile without AE3

Reply:
Restored. FVDelay will be implemented in separated ticket - it's a bigger change - for now A9 in FV will be disabled.